### PR TITLE
add group user column for view member

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Hiển thị user group cho view project member

## Evidence (Screenshot or Video)
[Screencast from 05-10-2023 14:21:24.webm](https://github.com/awesome-academy/project_management/assets/142207375/fa6becd1-a532-4762-b7fb-79c64a18adb8)

### CODE REVIEW

File: app/views/projects/_project_member.html.erb [hunk 1]
```diff
@@ -13,6 +13,7 @@
         <th width="5%"><%= t ".stt" %></th>
         <th><%= t ".Member_name" %></th>
         <th><%= t ".Role" %></th>
+        <th><%=t ".group"%></th>
```
Comment (betty2310) [inline app/views/projects/_project_member.html.erb]:
```suggestion
        <th><%= t ".group"%></th>
```

**Comment by betty2310** [comment]
okeee

**Comment by ghost** [comment]
ready

File: app/views/project_users/_project_user.html.erb [hunk 2]
```diff
@@ -2,6 +2,9 @@
   <td><%= sequence_number project_user_counter, @pagy&.offset %></td>
   <td><%= project_user.user_name %></td>
   <td><%= project_user.project_role_name %></td>
+  <td class="text-truncate" data-toggle="tooltip" data-placement="left" title="<%= project_user.group_name_text%>" style="cursor: pointer;">
+    <%= project_user.group_name_text%>
```
Comment (ngocvt-0484) [inline app/views/project_users/_project_user.html.erb]:
```suggestion
    <%= project_user.group_name_text %>
```

Comment (ngocvt-0484) [reply app/views/project_users/_project_user.html.erb]: để text dài chút dc k? nếu nhiều group thì hiển thị xuống dòng đi e
vd:
group A,
group B,
group C

Comment (ngocvt-0484) [reply app/views/project_users/_project_user.html.erb]: k cần trunicate

**Comment by ghost** [comment]
ready

**Pull request merged**